tests/commandLine.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	'basic functionality': browser => {
+		browser
+			.url('https://www.reddit.com/wiki/pages/')
+			.waitForElementVisible('#RESSettingsButton')
+			.keys([browser.keys['.']])
+			.assert.visible('#keyCommandLineWidget')
+			.keys([browser.keys.SHIFT, browser.keys.SLASH]) // "?"
+			.assert.containsText('#keyCommandForm', 'navigates to subreddit')
```

Comment:
That'll fail when it gets i18n'd.  How about some command name, or bullet points or something?